README.md

```diff
@@ -77,6 +84,21 @@ http client to access the service and it's api.
                 "delayFactor": 2,
                 // if the delay multiplied with the delay factor reaches maxDelay, the discovery will be aborted (default: 30000)
                 "maxDelay": 30000,
+                // models
+                "models": {
+                    // legacy format, will be transformed to { expose: true, isGlobal: true, isPublic: false }
+                    "ModelName1": true,
+                    // legacy format, will be transformed to { expose: false, isGlobal: false, isPublic: false }
+                    "ModelName2": false,
+                    "ModelName3": {
```

Comment:
May we should add the default value of the parameters in case only one or two of them are set.

Reply:
Just came into my mind. What happens if I do something like 
```
         "expose": false,
         "isGlobal": false
         "isPublic": true,
```

Reply:
I am totally aware of this. It will not be public since it is not propagated to the app. Since we already spent a lot of time for this package I do not really have time to cover everything.